nautobot.code-workspace

```diff
@@ -7,16 +7,10 @@
 	"extensions": {
 		"recommendations": [
 			"ms-python.python",
-			"esbenp.prettier-vscode",
-			"charliermarsh.ruff",
-			"dbaeumer.vscode-eslint"
+			"charliermarsh.ruff"
 		]
 	},
 	"settings": {
-		"eslint.format.enable": true,
-		"editor.codeActionsOnSave": {
-			"source.fixAll.eslint": true
-		},
 		"python.formatting.provider": "black",
 		"python.linting.enabled": true,
 		"python.linting.flake8Enabled": true,
```

Comment:
Looks like we missed some of these in the ruff PR. If you want to fix it here, it should look like this:

```json
    "[python]": {
        "editor.formatOnSave": true,
        "editor.defaultFormatter": "charliermarsh.ruff"
    },
```